_posts/2015-12-09-docker-for-development.markdown

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: Using Docker for Development
+date: 2015-12-09T00:00:00.000Z
+comments: false
+categories: [Docker, development]
+author: barry
+---
+
+When I first proposed using Docker for development, and began doing my work that way, there were some doubts.
+
+* Doesn't it seem like a lot of trouble to set up Docker to get my work done? 
+* Isn't it easier to use [Homebrew](http://brew.sh/) to install the services and database servers I need?
+
+<!-- more -->
+
+At Artsy, our main API aka Gravity uses MongoDB, Solr, Elasticsearch, and memcached. In development, we use [Mailcatcher](http://mailcatcher.me/) so we can view emails. When a new software engineer starts, that person studies a big Getting Started document, and spends most of a day to get everything installed and configured. Not only do they need to get the software installed, figuring out all of the environment variables that need to be set up can take some time too. While we have good documentation, it is still a tedious process. 
```

Comment:
You could mention that it's not only tedious, but repetitive and time-consuming for everyone (since the beginner needs to ask a lot of questions to get things set up).